### PR TITLE
(MAINT) merge master down to stable

### PR DESCRIPTION
**Pull request by jpinsonault**
In preparation for 2.7.0 release

### CODE REVIEW

**Comment by jpinsonault** [comment]
ping @cprice404 @camlow325. Merge was a fast forward, so not a whole not to review I imagine

File: project.clj [hunk 1]
```diff
@@ -1,8 +1,4 @@
-(def clj-version "1.7.0")
-(def tk-version "1.5.0")
-(def tk-jetty-version "1.5.10")
-(def ks-version "1.3.1")
-(def ps-version "2.6.1-stable-SNAPSHOT")
+(def ps-version "2.6.1-master-SNAPSHOT")
```
Comment (cprice404) [inline project.clj]: I think you do want to change this to 'stable', regardless of whether we do anything with the numbers yet.

**Comment by cprice404** [comment]
+1 other than changing the branch name in the version string

**Comment by cprice404** [comment]
if you want to change  it to 2.7.0-stable while you're in there, that'd be fine too

**Comment by jpinsonault** [comment]
@cprice404 updates the version to 2.7.0-stable-SNAPSHOT

**Pull request merged**